services/one-stream/package/buildAnyPackage.js

```diff
@@ -211,23 +211,6 @@ export const buildAnyPackage = async (packageId, config) => {
       putParams.Item.GSI1sk = packageId;
     } else {
       console.log("%s is not a OneMAC package: ", packageId, putParams.Item);
-      // if there was a package, and should not be now, delete the package item
-      if (currentPackage) {
-        const deleteParams = {
```

Comment:
we might have relied on this behavior in admin functions for when we did a change id
this is just a note to self to double check on this and if any changes would be needed in that regard.

Reply:
checked this and we do the delete of the package in the change id function and do not rely upon this here, which is good!